tests/pytest.ini

```diff
@@ -2,3 +2,4 @@
 log_level = DEBUG
 log_format = %(asctime)s.%(msecs)03d   %(levelname)s   %(message)s
 log_cli_format = %(asctime)s.%(msecs)03d   %(levelname)s   %(message)s
+addopts = -log-cli=true --log-cli-level=INFO
```

Comment:
Can this be overridden?

Would it be better to add:
log_cli = true
log_cli_level = INFO

Or are these equivalent?